src/pipecat/services/gladia.py

```diff
@@ -163,40 +140,37 @@ async def run_stt(self, audio: bytes) -> AsyncGenerator[Frame, None]:
         await self.stop_processing_metrics()
         yield None
 
-    async def _setup_gladia(self):
-        configuration = {
-            "x_gladia_key": self._api_key,
-            "encoding": "WAV/PCM",
-            "model_type": "fast",
-            "language_behaviour": "manual",
-            "sample_rate": self._settings["sample_rate"],
-            "language": self._settings["language"],
-            "transcription_hint": self._settings["transcription_hint"],
-            "endpointing": self._settings["endpointing"],
-            "prosody": self._settings["prosody"],
-        }
-
-        await self._websocket.send(json.dumps(configuration))
+    def _init_live_session(self):
+        response = requests.post(
```

Comment:
this should be using `aiohttp`. we should pass the session in the constructor as in other services. `requests` would block the asyncio event loop.